src/components/Questionnaire/show.tsx

```
@@ -0,0 +1,225 @@
+      //     responses: values.map((v) => ({
+      //       question_id: v.id,
+      //       value: v.value,
+      //       note: v.note,
+      //       bodysite: v.bodysite,
+      //       method: v.method,
+      //     })),
+      //   },
+      // });
+    } catch (error) {
+      console.error("Failed to submit form:", error);
+    }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)